spec/models/shipping_location_spec.rb

```diff
@@ -2,11 +2,48 @@
 require 'shipping_location'
 
 RSpec.describe ShippingLocation do
+  let(:valid_loc) { ShippingLocation.new(city: "Beaverton", state: "OR", zip: 97005, country: "USA")}
+  let(:invalid_loc) { ShippingLocation.new }
```

Comment:
I like to keep the scope of these as limited as possible -- will we use invalid_loc outside of the validations describe block?